resources/models/availability.py

```diff
@@ -142,14 +142,17 @@ def _check_closed(self):
     def clean(self, ignore_overlap=False):
         super(Period, self).clean()
 
+        if self.start is None or self.end is None:
+            raise ValidationError(_("You must set 'start' and 'end' fields."), code="empty_start_end")
```

Comment:
Hmm.. since the `start` and `end` fields don't have `blank=True`, shouldn't they already be required in forms? See the [docs of blank attribute](https://docs.djangoproject.com/en/1.11/ref/models/fields/#blank):
> If a field has blank=True, form validation will allow entry of an empty value. If a field has blank=False, the field will be required.